nexus/value/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 use array::ArrayValue;
+use base64::Engine;
```

Comment:
What do you think about [something like](https://docs.rs/base64/latest/base64/prelude/index.html):

```
use base64::prelude::{Engine as _, BASE64_STANDARD}
```

We can then use `BASE64_STANDARD`.